Gruntfile.js

```diff
@@ -146,7 +146,10 @@ module.exports = function (grunt) {
      */
     grunt.registerTask('test:unit', function(app) {
         var target = app ? ':' + app : '';
-        grunt.config.set('karma.options.singleRun', (options.singleRun === false) ? false : true);
+        if (options.singleRun === false) {
+            grunt.config.set('karma.options.singleRun', false);
+            grunt.config.set('karma.options.autoWatch', true);
+        }
```

Comment:
It's somehow related. It was the only way to easily run unit tests while developing this feature.